Remove footnotes from the language reference guide (#847)
Dyalog/documentation: language-reference-guide/docs/primitive-operators/i-beam/create-data-binding-source.md

    **.NET Framework only**

> Creates an object that may be used as a data source for WPF data binding. See Microsoft Developer Network, Data Binding Overview for a full explaination of the concepts of WPF data binding.

Reviewer: ```suggestion
Creates an object that may be used as a data source for WPF data binding. For more information on the concepts of WPF data binding, see [Microsoft Developer Network's Data Binding Overview](https://learn.microsoft.com/en-us/dotnet/desktop/wpf/data/).
```
